src/Processors/Formats/Impl/RawBLOBRowOutputFormat.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <Core/Block.h>
+#include <Processors/Formats/IRowOutputFormat.h>
+
+
+namespace DB
+{
+
+class WriteBuffer;
+
+
+/** This format only allows to output one column of type String or similar.
+  * It is output as raw bytes without any delimiters or escaping.
+  *
+  * The difference between RawBLOB and TSVRaw:
+  * - only single column dataset is supported;
```

Comment:
Actually multiple column dataset is also supported (all values will be output without any separators).